### DISCUSSION TITLE
xsd types genereated by the Rio-Parser

### DISCUSSION THREAD

**Discussion by naturzukunft** [eclipse-rdf4j/rdf4j#5217] [Q&A]
Hi there,
when i'm parsing a json-ld file 
```
RDFParser parser = Rio.createParser(format);
model = new LinkedHashModel();
parser.setRDFHandler(new StatementCollector(model));
parser.getParserConfig().set(JSONLDSettings.SECURE_MODE, false);
parser.parse(new StringReader(graphAsString));
```
The datatype for dateTime ist `http://www.w3.org/2001/XMLSchema#dateTime`
What's todo, if i wish to have `https://www.w3.org/TR/xmlschema11-2/#dateTime`

**Comment by hmottestad** [comment]
You could try to implement your own RDFHandler and transform the data at that point. 

I don't think that the JSON-LD parser does anything to infer datatypes. Does the data have a different datatype than what is produced by the parser?
